data/registry/application-integration-collector-cloud foundry.yml

```diff
@@ -1,7 +1,7 @@
 title: Cloud Foundry
 registryType: application integration
 language: collector
-tags:
+tags: Metrics
   - collector
   - cloud foundry
```

Comment:
"collector" and "cloud foundry" are the tags already, you don't need to add anything to those files.